index.bs

```diff
@@ -398,6 +401,8 @@ Otherwise, do nothing.
 <emu-alg>
   1. Let _called_ be the number of times _count_ has been invoked (including this invocation) with
   the provided _label_.
+    1. If the associated <a>count map</a> contains an entry with key _label_, <a for="map">set</a> the _value_ of map[_label_] to be map[_label_] + 1.
+    1. Else, <a for="map">set</a> the _value_ of the entry with key _label_ in the associated <a>count map</a> to be 1.
```

Comment:
Nit: Else => Otherwise

Probably not super important but I see If/Otherwise used a lot in the spec(s).